### PR TITLE
feat(drupal-attributes): Refactored select, input, form with drupal attributes - FRONT-2428

### PR DESCRIPTION
**Pull request by tibi2303**

### CODE REVIEW

File: src/components/form-input/form-input.story.js [hunk 1]
```diff
@@ -42,7 +42,21 @@ const getArgTypes = (data, type) => {
   };
```
Comment (planctus) [inline src/components/form-input/form-input.story.js:42]: can we fix the toggle_variant control..? Should have the default value` 'primary' || data.variant`.. ;)

**Review by planctus — APPROVED** [review]

File: src/components/form-input/form-input.story.js (hunk 1 above)
Comment (tibi2303) [reply src/components/form-input/form-input.story.js:42]: done

File: src/components/form/form.html.twig [hunk 2]
```diff
@@ -23,12 +23,9 @@
```
Comment (planctus) [inline src/components/form/form.html.twig:23]: can you remove the extra_classes also from the doc comment in the template?

Comment (tibi2303) [reply src/components/form/form.html.twig:22]: done

**Review by planctus — APPROVED** [review]

**Pull request merged**